locustfile.py

```diff
@@ -164,6 +171,17 @@ def upload(self):
         if form:
             self.upload_addon(form)
 
+    @task(5)
+    def browse(self):
+        self.client.get('/en-US/firefox/')
+        self.client.get('/en-US/firefox/search/?q=pi&appver=45.0&platform=mac')
+        with self.client.get(
+                '/en-US/firefox/extensions/',
+                allow_redirects=False, catch_response=True) as response:
```

Comment:
when you use `catch_response` I think you have to call `response.failure()` manually otherwise it won't ever happen.